vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -598,6 +598,34 @@ function Exclude-ReservedUDPSourcePort()
     Invoke-Executable -Executable "netsh.exe" -ArgList @("int", "ipv4", "add", "excludedportrange", "udp", "65330", "1", "persistent")
 }
 
+function Get-LatestWindowsDefenderUpdate {
```

Comment:
This is to get windows defender platform update. Suggest adding `platform` in the pr title, description and code.